src/Jecoute/FileStatsService.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace AppBundle\Jecoute;
+
+use AppBundle\Entity\Jecoute\Survey;
+use AppBundle\Entity\Jecoute\SurveyQuestion;
+use AppBundle\Repository\Jecoute\DataAnswerRepository;
+
+class FileStatsService
+{
+    private $dataProvider;
+    private $dataAnswerRepository;
+    private $content;
+
+    public function __construct(StatisticsProvider $provider, DataAnswerRepository $dataAnswerRepository)
+    {
+        $this->dataProvider = $provider;
+        $this->dataAnswerRepository = $dataAnswerRepository;
+    }
+
+    public function getStatsFileCsvBySurvey(Survey $survey): array
```

Comment:
What do you think to rename this method by `export()` ?

Reply:
renamed